lib/rails/perftest/railties/testing.tasks

```diff
@@ -4,14 +4,25 @@ namespace :test do
   task 'perftest:benchmark_mode' do
     ENV["BENCHMARK_TESTS"] = '1'
   end
+  if Rake::Task.task_defined?('test:prepare')
```

Comment:
should we still have the fallback to db:test:prepare or is there something that you would prefer instead?